server/plugin.go

```diff
@@ -53,6 +55,13 @@ type Plugin struct {
 	configuration *configuration
 }
 
+// New returns an instance of a Plugin
+func New() *Plugin {
+	return &Plugin{
+		githubPermalinkRegex: regexp.MustCompile(`https?://(?P<haswww>www\.)?github\.com/(?P<user>[\w-]+)/(?P<repo>[\w-]+)/blob/(?P<commit>\w+)/(?P<path>[\w-/.]+)#(?P<line>[\w-]+)?`),
```

Comment:
Would be good to doc what does this match.

Reply:
I thought the name was clear in expressing that it matches a github permalink. What do you have in mind ?